runtime/sam/expr/function/ztests/strftime.yaml

```
@@ -0,0 +1,11 @@
+zed: 'strftime(f, v)'
+
+input: |
+  {f:"%Y-%a",v:2024-07-30T06:15:01.062681Z}
+  {f:1,v:2024-07-30T06:15:01.062681Z}
```

Nit: Test a malformed format string.
```suggestion
  {f:1,v:2024-07-30T06:15:01.062681Z}
  {f:"%",v:2024-07-30T06:15:01.062681Z}
```